moveit_setup_assistant/package.xml

```diff
@@ -22,13 +22,13 @@
   <build_depend>qtbase5-dev</build_depend>
   <build_depend>libqt5-opengl-dev</build_depend>
   <build_depend>ompl</build_depend>
+  <build_depend>rviz</build_depend>
 
   <depend>moveit_core</depend>
   <depend>moveit_ros_planning</depend>
   <depend>moveit_ros_visualization</depend>
   <depend>rosconsole</depend>
   <depend>roscpp</depend>
-  <depend>rviz</depend>
```

Comment:
`<depend>` [adds build_depend,exec_depend, and build_export_depend](https://wiki.ros.org/catkin/package.xml).

build_export_depend is the offender here, the other two should still be required. So please add `<exec_depend>` too.

There was a reason why package.xml version 1 did not allow this kind of "generic depend"...